bot/cogs/watchchannels/bigbrother.py

```diff
@@ -111,8 +120,13 @@ def __init__(self, bot: Bot) -> None:
 
             await post_infraction(ctx, user, 'watch', f"Unwatched: {reason}", hidden=True, active=False)
 
-            await ctx.send(f":white_check_mark: Messages sent by {user} will no longer be relayed.")
+            if not banned:  # Prevents a message being sent to the channel if part of a permanent ban
+                log.trace("User is not banned.  Sending message to channel")
+                await ctx.send(f":white_check_mark: Messages sent by {user} will no longer be relayed.")
 
             self._remove_user(user.id)
         else:
-            await ctx.send(":x: The specified user is currently not being watched.")
+            log.trace("No active watches found for user.")
+            if not banned:  # Prevents a message being sent to the channel if part of a permanent ban
+                log.trace("User is not perma banned. Send the error message.")
+                await ctx.send(":x: The specified user is currently not being watched.")
```

Comment:
I think this will be cleaner if the message is assigned to a parameter and sent at the very end, similar to what the `apply_watch` function does.

Reply:
Addressed in most recent commit